### PR TITLE
fix: video toggle disabled in MLS calls - WPB-6204

### PR DESCRIPTION
**Pull request by netbe**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-6204" title="WPB-6204" target="_blank">BugWPB-6204</a>  [iOS] MLS: Cannot turn on camera when joining video call on bund-next
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

On a MLS conference calls, sometimes the video button is disabled

### Causes (Optional)

When receiving the call, we set the wrong value of `isConferenceCall`  to the callSnapshot. We check only for conference value and not mlsConference. This results on caller being able to enable the video but not receivers.

### Solutions

Use helper method from AVSIdentiferType to get the right value of `isConferenceCall`

#### Test Coverage (Optional)

- [x] I have added automated test to this contribution

#### How to Test

- MLS Call with multiple people
- answer on iphone
- you should be able to enable video

### CODE REVIEW

**Review by caldrian — APPROVED** [review]

File: wire-ios-sync-engine/Source/Calling/CallCenterSupport.swift [hunk 1]
```diff
@@ -43,6 +43,19 @@ public enum AVSConversationType: Int32 {
 
 }
 
+extension AVSConversationType {
```
Comment (markusfassbender) [inline wire-ios-sync-engine/Source/Calling/CallCenterSupport.swift]: Suggestion: move this extension to a file `AVSConversationType+Conference.swift`. It makes it easier to discover in the project and the files smaller.

**Review by markusfassbender — APPROVED** [review]

**Review by theswiftway — APPROVED** [review]
LGTM

**Comment by theswiftway** [comment]
**question:** Not related but I saw that `AVSConversationType` & `AVSCallType` are explicitly numbered. They would be numbered the same even without that. Is it for any particular reason?

**Comment by netbe** [comment]
> **question:** Not related but I saw that `AVSConversationType` & `AVSCallType` are explicitly numbered. They would be numbered the same even without that. Is it for any particular reason?

@theswiftway yes you're right, maybe at the time they just ported to objc without having this in mind

**Pull request merged**